.travis.yml

```diff
@@ -8,6 +8,9 @@ python:
 addons:
   apt:
     packages:
-    - rpm
+      - rpm
+  pip:
```

Comment:
Seems like travis ignores this. ;(

Reply:
I don't think there is a `pip` addon?

Reply:
twas worth a try. I woulda expect it to fail, though